src/fsharp/FSComp.txt

```diff
@@ -1533,4 +1533,5 @@ forFormatInvalidForInterpolated4,"Interpolated strings used as type IFormattable
 3380,parsEofInInterpolatedVerbatimString,"Incomplete interpolated verbatim string begun at or before here"
 3381,parsEofInInterpolatedTripleQuoteString,"Incomplete interpolated triple-quote string begun at or before here"
 3382,lexRBraceInInterpolatedString,"A '}}' character must be escaped (by doubling) in an interpolated string."
+featureOverloadsForCustomOperations,"overloads for custom operations"
```

Comment:
I believe line around 1514 would be better to group the features together

Reply:
I moved it further up next to the line that says overloads are not allowed.